Sources/AudioKitUI/Helpers/Helpers.swift

```diff
@@ -0,0 +1,11 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKitUI/
+
+import Foundation
+
+func map(n: Double, start1: Double, stop1: Double, start2: Double, stop2: Double) -> Double {
```

Comment:
I don't think these global map functions are what we want to use. We have some extensions in AudioKitHelpers.swift in AudioKit that normalize and denormalize to a range.  You could either reappropriate those or use them as models for extension to Double and CGFloat.  Something like 'value.mapped(to: 0...1000, from: 0...1)` ie. closed ranges. Ranges are better than these stop and start values, which by the way are verbs and don't read that well as is.

Reply:
I really like this suggestion - cleans things up a lot. I added these methods to AudioKitHelpers. This PR is now dependent on this:
https://github.com/AudioKit/AudioKit/pull/2414